public/uploadStudentList.php

```diff
@@ -59,6 +81,17 @@
     <input size='50' type='file' name='filename' accept=".csv">
     </br>
 
+    <!-- TO DO change to add info on course name as well -->
+    <label for="section">Select Course section</label>
+    <select Name="section" id="section">
+            <option value="">----Select----</option>
```

Comment:
try this instead 
`<option value="" disabled selected>----Select----</option>`

Reply:
done